spec.bs

```diff
@@ -441,8 +441,9 @@ To <dfn>build bid generators map</dfn> given an [=auction config=] |auctionConfi
 
 <div algorithm>
 
-To <dfn>generate and score bids</dfn> given an [=auction config=] |auctionConfig|, a
-[=global object=] |global|, and a [=boolean=] |isTopLevel|:
+To <dfn>generate and score bids</dfn> given an [=auction config=] |auctionConfig|, an
+[=auction config=] |topLevelAuctionConfig|, a [=global object=] |global|, and a [=boolean=]

```

Comment:
It's not really valid to pass `null` as an argument for this parameter as you do further above, so I'd make this `, an [=auction config=]-or-null |topLevelAuctionConfig|` and then null-check it later. That would be more correct, and gives you the ability to use the null-ness of that variable to check if this is a top-level instantiation, so you can remove the `|isTopLevel|` boolean param.

Reply:
Fixed, thanks!